.github/workflows/deploy-docs.yml

```
@@ -181,11 +195,12 @@
       # 💬 Leave (or update) a sticky comment on the PR with the preview link
       - name: 💬 Comment PR with preview URL
         if: ${{ steps.find_pr.outputs.pr_number != '' }}
-        uses: marocchino/sticky-pull-request-comment@v2
+        uses: ./internals/c15t-github-action
```

actionlint (1.7.7)

missing input "GITHUB_TOKEN" which is required by action "C15T Sticky Pull Request Comment" defined at "./internals/c15t-github-action". all required inputs are "GITHUB_TOKEN"

(action)